### DISCUSSION TITLE
HTTP ERROR 500 - SETELAH DI UPLOAD KE HOSTING

### DISCUSSION THREAD

**Discussion by mohamadlukman** [kirimwa-id/forum#141] [Tanya Jawab]
ketika bukan scan qr muncul error **HTTP ERROR 500**
Versi php: 7.4.8 

untuk scrip **scan-qr.php**

`<?php
require 'functions.php';

try {
  $query = http_build_query(['device_id' => 'henpon-01']);
  $baseApiUrl = getenv('API_BASEURL') ? getenv('API_BASEURL') : 'https://api.kirimwa.id/v1';
  $reqParams = [
    'token' => 'Sudah Di masukan API WA',
    'url' => sprintf('%s/qr?%s', $baseApiUrl, $query)
  ];

  $response = apiKirimWaRequest($reqParams);
  echo $response['body'];
} catch (Exception $e) {
  print_r($e);
}`

untuk edd-device berhasil
Mohon infonya knp ya?

**Comment by kafaa-dev** [comment]
Coba cek log error di hostingnya mas

**Reply by mohamadlukman** [reply]
gimna caranya ya?

**Reply by mohamadlukman** [reply]
soalnya pake hosting free..hihi

**Reply by kafaa-dev** [reply]
hmm, coba nyalain display error nya sementara dulu

**Reply by mohamadlukman** [reply]
cara nyalain nya gmn mas?

**Reply by kafaa-dev** [reply]
hosting apa ya?
